modules/extension/wmts/src/main/java/org/geotools/ows/wmts/request/GetTileRequest.java

```diff
@@ -42,6 +42,14 @@
 
     void setFormat(String format);
 
+    void setTileMatrixSet(String tileMatrixSet);
```

Comment:
This interface is not deprecated and can now play the role of both single and multi-tile request? Seems inconsistent, one  should probably either use xyz or bbox no?

Reply:
@aaime I was wondering about deprecating those properties (bbox) at a later stage. My intention is to work more on how WMTSCoverageReader fetches the tiles. I've noticed that someone else is working on that class with GEOT-7036, so I would wait with too much changes. After changing WMTSCoverageReader it will be a lot more easier to deprecate.